models/addresses/base/addresses_base_dex.sql

```diff
@@ -0,0 +1,30 @@
+{{config(alias = 'dex',
+        tags=['static'],
+        post_hook='{{ expose_spells(\'["base"]\',
```

Comment:
does this need to be exposed on the app, or is it more useful for downstream spells? we're trying to clean up the usage of the data explorer a bit

Reply:
I'm good without exposing this and only using for downstream spells

Reply:
sounds good. once you resolve these two comments, we can look to proceed